Frontend/Pages/Components/AnswerComponent.razor

```diff
@@ -0,0 +1,80 @@
+﻿@using Models
+@using System.Web
+@using System.Net
+
+<li class="list-group-item px-0">
+    <div class="row">
+        <div class="col-auto">
+            <h6 class="fw-bold mt-1">Antwort:</h6>
+        </div>
+        <div class="col">
+            <h6 class="fw-bold mt-1 mb-1">
+                @AnswerCreator.FirstNames @AnswerCreator.LastName
+                <small class="text-muted">@Answer.CreationTime</small>
+            </h6>
+            @Answer.Text
+        </div>
+    </div>
+    @if (Answer.CreatorId == _userId)
+    {
+        <div class="d-grid mt-2 d-flex justify-content-end">
+            <button class="btn btn-outline-danger btn-small" type="button" @onclick="() => OpenDeleteAnswerDialog(Question.Id, Answer.Id)">
+                Löschen
+            </button>
+        </div>
+    }
+</li>
+
+@if (DeleteAnswerDialogOpen) {
+    <Frontend.Pages.Components.DeleteModal 
+    Title="Antwort löschen" 
+    Text="Sind Sie sicher, dass Sie Ihre Antwort löschen möchten?" 
+    ToDelete="answer" 
+    WseId="WseId"
+    QuestionId="QuestionIdToDelete" 
+    AnswerId="AnswerIdToDelete"
+    OnClose="@OnDeleteAnswerDialogClose"/>
+}
+
+@code {
+    [Inject]
+    private HttpClient HttpClient { get; set; } = default!;
+
+    [Parameter]
+    public Question Question { get; set; } = null!;
```

Comment:
Inconsistent, either we're going with `default!` or with `null!`. Idc which one, but not both.